llama.cpp

```diff
@@ -1034,7 +1034,7 @@ struct llama_mmap {
         int fd = fileno(file->fp);
         int flags = MAP_SHARED;
         // prefetch/readahead impairs performance on NUMA systems
-        if (numa) { prefetch = 0; }
+        if (numa > 0) { prefetch = 0; }
```

Comment:
The parameter `numa` is still a bool, so this change is unnecessary. The value of this parameter is obtained from `ggml_is_numa`.